### PR TITLE
Updates for PureScript 0.15

### PR DESCRIPTION
**Pull request by thomashoneyman**
This makes a number of updates so the library will be compatible with all the other PureScript 0.15 library releases.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,17 @@
-language: node_js
+4anguage: node_js
```
Comment (fsoikin) [inline .travis.yml:1]: Typo?

File: Makefile [hunk 2]
```diff
@@ -21,16 +21,16 @@ ctags:
 		> tags
 
 build-example: $(OUTPUT)
-	pulp build -I example --to $(OUTPUT)/example.js
+	npx spago bundle-app --path $(example) --to $(OUTPUT)/example.js --minify
```
Comment (fsoikin) [inline Makefile:24]: Isn't bundling functionality going away?

File: .travis.yml (hunk 1 above)
Comment (thomashoneyman) [reply .travis.yml]: Yes! Typo.

File: Makefile (hunk 2 above)
Comment (thomashoneyman) [reply Makefile:24]: Native bundling is going away, and instead Spago will call out to esbuild under the hood (if it's installed), or you can swap in another bundler.

**Comment by thomashoneyman** [comment]
@fsoikin I seem to have busted something related to the `ansi` dependency, but it's quite late and I don't have any more time today. I'll come back tomorrow to see about getting this building!

File: bower.json [hunk 3]
```diff
@@ -15,17 +15,17 @@
...
-    "purescript-now": "^5.0.0"
+    "purescript-avar": "^5.0.0",
+    "purescript-console": "^6.0.0",
+    "purescript-aff": "^7.0.0",
+    "purescript-exceptions": "^6.0.0",
+    "purescript-strings": "^6.0.0",
+    "purescript-prelude": "^6.0.0",
+    "purescript-transformers": "^6.0.0",
+    "purescript-foldable-traversable": "^6.0.0",
+    "purescript-pipes": "https://github.com/thomashoneyman/purescript-ansi.git#trh/purs-0.15",
```
Comment (fsoikin) [inline bower.json:26]: I would venture this line is at fault :-)

```suggestion
    "purescript-pipes": "https://github.com/thomashoneyman/purescript-pipes.git#trh/purs-0.15",
```

Comment (thomashoneyman) [reply bower.json]: Ha! Ok, that wasn’t too bad 😅

**Review by fsoikin — APPROVED** [review]
Looks good to me!

**Comment by fsoikin** [comment]
Cut and published v7.0.0, thanks @thomashoneyman!

**Pull request merged**